### ISSUE TITLE
Docs questions/improvements

### ISSUE THREAD

**Issue by benjiallen** [smockle/action-transfer-issues#36]
As i'm reading the instructions, I have a couple of questions. Maybe these could be considered docs improvements.

1. Does it matter where I run the action? Does the action need to be run from the source or the destination repo?
2. The environment variable used in the example is `secrets.GH_TOKEN` but when reading [Automatic token authentication](https://docs.github.com/en/actions/security-guides/automatic-token-authentication) it looks like the default name for this token is `GITHUB_TOKEN`. Which one should I use? Does it even matter?

**Comment by benjiallen** [comment]
One other big learning - we might want to mention rate limits in the docs 😄 Adding here so I don't forget.

**Comment by smockle** [comment]
> 1. Does it matter where I run the action? Does the action need to be run from the source or the destination repo?

Nope! The workflow which runs this action can be located in the source repo, the destination repo, or neither. The provided token must have access to both repos; the placement of the workflow file itself is incidental.

> 2. The environment variable used in the example is `secrets.GH_TOKEN` but when reading [Automatic token authentication](https://docs.github.com/en/actions/security-guides/automatic-token-authentication) it looks like the default name for this token is `secrets.GITHUB_TOKEN`. Which one should I use? Does it even matter?

`secrets.GITHUB_TOKEN` is automatically created and included in the GitHub Actions runtime environment, but it’s typically not suitable for use with `smockle/action-transfer-issue`, because it is scoped to the org+repo where the workflow is run. In contrast, a Personal Access Token (PAT) represents your user; it can have broader access, including across two orgs, making it useful for transfer operations.

**Comment by smockle** [comment]
> One other big learning - we might want to mention rate limits in the docs 😄 Adding here so I don't forget.

I’ve transferred ~100 issues in a single workflow run without hitting a rate limit, but more recently, I’ve seen secondary rate limits hit within 20 issues. I’d like to figure out why I’m seeing different behavior now—before changing the code (e.g. adding a sleep) or docs—though I’m open to changing both!

**Comment by benjiallen** [comment]
I've added a todo. I'd like to contribute back to the docs based on your responses here. PR should come later this week.
